specification/batch/data-plane/Microsoft.Batch/stable/2018-12-01.8.0/BatchService.json

```diff
@@ -0,0 +1,15868 @@
+                "description": "The client-request-id provided by the client during the request. This will be returned only if the return-client-request-id parameter was set to true.",
+                "type": "string",
+                "format": "uuid"
+              },
+              "request-id": {
+                "description": "A unique identifier for the request that was made to the Batch service. If a request is consistently failing and you have verified that the request is properly formulated, you may use this value to report the error to Microsoft. In your report, include the value of this request ID, the approximate time that the request was made, the Batch account against which the request was made, and the region that account resides in.",
+                "type": "string",
+                "format": "uuid"
+              },
+              "ETag": {
+                "description": "The ETag HTTP response header. This is an opaque string. You can use it to detect whether the resource has changed between requests. In particular, you can pass the ETag to one of the If-Modified-Since, If-Unmodified-Since, If-Match or If-None-Match headers.",
+                "type": "string"
+              },
+              "Last-Modified": {
+                "description": "The time at which the resource was last modified.",
+                "type": "string",
+                "format": "date-time-rfc1123"
+              }
+            },
+            "description": "A response containing the certificate.",
+            "schema": {
+              "$ref": "#/definitions/Certificate"
+            }
+          },
+          "default": {
+            "description": "The error from the Batch service.",
+            "schema": {
+              "$ref": "#/definitions/BatchError"
+            }
+          }
+        }
+      }
+    },
+    "/jobs/{jobId}/tasks/{taskId}/files/{filePath}": {
+      "delete": {
+        "tags": [
+          "Files"
+        ],
+        "operationId": "File_DeleteFromTask",
+        "x-ms-examples": {
+          "File delete from task": { "$ref": "./examples/FileDeleteFromTask.json" }
+        },
+        "summary": "Deletes the specified task file from the compute node where the task ran.",
+        "x-ms-request-id": "request-id",
+        "parameters": [
+          {
+            "name": "jobId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The ID of the job that contains the task."
+          },
+          {
+            "name": "taskId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The ID of the task whose file you want to delete."
+          },
+          {
+            "name": "filePath",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The path to the task file or directory that you want to delete."
+          },
```

Comment:
@matthchr - Matt, sorry, can you please update this description to change the incorrect doc link? I suggest: 
 "description": "This property is mutually exclusive with other ImageReference properties. The virtual machine image must be in the same region and subscription as the Azure Batch account. For more details, see https://docs.microsoft.com/azure/batch/batch-custom-images."

Reply:
@dlepow did you put this on the wrong line?

Reply:
I think I found where it was meant to be though -- I've fixed the docs there

Reply:
@matthchr - Thanks. When I commented before the GitHub UI was a bit flaky with the large file, so I might have selected the wrong place.